components/rise/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/rise",
-  "version": "0.0.2",
+  "version": "0.0.1",
```

Because of some issues we had, the latest version published is `0.0.2`, so this will have to be `0.0.3`

```suggestion
  "version": "0.0.3",
```